homeassistant/components/deconz/climate.py

```diff
@@ -231,16 +245,20 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
     @property
     def current_temperature(self) -> float:
         """Return the current temperature."""
-        return self._device.temperature
+        return self._device.temperature  # type: ignore[no-any-return]
```

Comment:
I really think this doesn't help.
I would strongly recommend on typing the library first.

Reply:
It will help because future PRs will be smaller (hopefully :) )

Reply:
I think it is the other way around. This (and the other PRs) are full of added complexity, workarounds and ignores comments, to workaround the fact the library isn't typed.

All of that wasn't needed if you did that first. So I'm on the other end and actually think you should stop this effort on this end until the library has been typed.

Reply:
I could remove the type ignore part, to keep it as close to the end result as possible?